synapse/axon.py

```diff
@@ -1487,6 +1523,70 @@ async def jsonlines(self, sha256, errors='ignore'):
                 raise s_exc.BadJsonText(mesg=f'Bad json line encountered while processing {sha256}, ({e})',
                                         sha256=sha256) from None
 
+    async def read(self, sha256, size, offset=0):
```

Comment:
Does this replicate the use of `Axon.get()` ? I think we can add the size constraints in the storm API, and then we don't have duplicate telepath APIs.